.github/workflows/main.yml

```diff
@@ -90,18 +94,25 @@ jobs:
     - name: Use strawberry perl
       if: startsWith(matrix.os, 'windows')
       run: echo OPENSSL_SRC_PERL=C:/Strawberry/perl/bin/perl >> $GITHUB_ENV
-    - run: |
+    - name: Run tests (not Windows)
+      if: "!startsWith(matrix.os, 'windows')"
+      run: |
         set -e
         cargo generate-lockfile
         ./ci/run-docker.sh ${{ matrix.target }}
-      if: "!startsWith(matrix.os, 'windows')"
-      name: Run tests (not Windows)
-    - run: |
+    - name: Download nasm.exe (Windows)
+      if: matrix.nasm_exe == 'installed'
+      run: |
+        WINNASMVERSION='2.15.05'
+        curl -O https://www.nasm.us/pub/nasm/releasebuilds/${WINNASMVERSION}/win64/nasm-${WINNASMVERSION}-win64.zip
+        unzip nasm-${WINNASMVERSION}-win64.zip
+        echo "$PWD/nasm-${WINNASMVERSION}" >> $GITHUB_PATH
```

Comment:
Could this step also set `OPENSSL_RUST_USE_NASM`?

Reply:
Leaving it unset means auto-detecting. Adding this makes it works better in CI -- if some one don't install nasm correctly, the panic will let us know.